CONTRIBUTING.md

```diff
@@ -2,13 +2,33 @@
 
 ## Table of Contents
 - [Claiming an Issue](#Claiming-an-Issue)
+- [step 1. create an issue](#step-1.-create-an-issue)
+- [step 2 fork the code](#step-2-fork-the-code)
+- [step 3 Make the necessary changes](#step-3-Make-the-necessary-changes)
+- [step 4 Submit a PR](#step-4-Submit-a-PR)
+- [CONGRADUALTIONS](#CONGRADUALTIONS)
```

Comment:
Don't include the minor paths in the table of contents... It will make the ToC harder to Read

```suggestion
```